### DISCUSSION TITLE
How to set contour lines to use metric units?

### DISCUSSION THREAD

**Discussion by petersohn** [osmandapp/OsmAnd#13819] [Q&A] [answered]
I have metric units (kilometers/meters) set in the settings for all profiles. Yet, numbers written on contour lines are displayed in feet. I cannot find any settings for this. The contour lines plugin doesn't even have a settings button. Is there any way to make it use meters?

**Comment by Lee-Carre** [comment]
See #12152 & #13117.

Apparently it's in the “Configure Map menu”.

Though, this raises a good point; that unit preference should have an option to inherit the value from the profile.

**Comment by petersohn** [comment]
I found no such option in Configure Map either. However, I found a solution. It can apparently be set when downloading the contour line files. So I deleted all contour lines and downloaded each of them again, now in meters.

**Reply by Lee-Carre** [reply]
From [another issue](/osmandapp/OsmAnd/issues/13818), you're using

    OsmAnd Version: OsmAnd+ 4.1.11

The referenced changes may have been committed after v4.1.11 was released.

They should be available in the *next* release.

I'm pleased that you found a workaround, though.

**Comment by dmpr0** [comment, marked as answer]
Contour lines metric units can be changed only by redownloading the file in the needed format.
